From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Mon, 11 May 2020 23:43:02 +0200
Subject: [PATCH] multipath-tools tests/test-lib.c: drop
 __wrap_is_claimed_by_foreign

is_claimed_by_foreign() is an inline function and can't be wrapped.

Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 tests/test-lib.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/tests/test-lib.c b/tests/test-lib.c
index 59275163..00bae58e 100644
--- a/tests/test-lib.c
+++ b/tests/test-lib.c
@@ -56,12 +56,6 @@ int __wrap_execute_program(char *path, char *value, int len)
 	return 0;
 }
 
-bool __wrap_is_claimed_by_foreign(struct udev_device *ud)
-{
-	condlog(5, "%s: %p", __func__, ud);
-	return false;
-}
-
 struct udev_list_entry
 *__wrap_udev_device_get_properties_list_entry(struct udev_device *ud)
 {
-- 
2.17.2

